content/en/blog/_posts/2019-05-22-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md

```diff
@@ -0,0 +1,194 @@
+
+---
+layout: blog
+title: "OPA Gatekeeper: Policy and Governance for Kubernetes"
+date: 2019-05-22
+slug: OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes 
+---
+
+**Authors:** Rita Zhang (Microsoft), Max Smythe (Google), Craig Hooper (Commonwealth Bank AU), Tim Hinrichs (Styra), Lachie Evenson (Microsoft), Torin Sandall (Styra)
+
+In this post, we will walk through the goals, history, and current state of the [Gatekeeper](https://github.com/open-policy-agent/gatekeeper) project, and how you can leverage it to help enforce policies and strengthen governance in your Kubernetes environment. 
```

Comment:
**Nitpick**: This block of text is a bit difficult for a general audience to understand and could be worded as follows to make it a bit more clear. 


The Gatekeeper project can be leveraged to help enforce policies and strengthen governance in your Kubernetes environment.

The following recordings from the Kubecon EU 2019 sessions are a great starting place in working with Gatekeeper: